lib/Doctrine/Common/Annotations/TokenParser.php

```diff
@@ -12,6 +12,8 @@
 use const T_AS;
 use const T_COMMENT;
 use const T_DOC_COMMENT;
+use const T_NAME_FULLY_QUALIFIED;
```

Comment:
These issues were in the repo... but I guess CI didn't catch them before? Weeeird

Reply:
Not on 1.11.x, though :laughing: